MetaMask/metamask-extension
# added AvatarAccount component

## Fixes #15084 

### After

<img width="1875" alt="Screenshot 2022-09-12 at 5 24 08 PM" src="https://user-images.githubusercontent.com/39872794/189646870-41a1d4fd-00d0-4b0d-9456-2c6ab3970971.png">

## Manual Testing Steps

Storybook
- Go to the latest build of storybook in this PR
- Search for AvatarAccount in component-library
- Check the stories, controls and documentation

## Pre-Merge Checklist


### + If there are functional changes:
